File: cfgs/java_cfgs_foo/ARRAY_RANGE_QUERIES_ELEMENTS_FREQUENCY_VALUE.java
Nodes
1;special;;;
2;statement;Map<Integer, Integer> mp = new HashMap<>();14;
3;statement;int i = foo_1;15;
4;conditional;foo_1 <= foo_2;16;
5;statement;fun_1(foo_2[foo_3], fun_2(foo_5[foo_6]) == null ? 1 : fun_3(foo_8[foo_9]) + 1);17;
6;statement;i++;17;
7;statement;int count = 0;18;
8;statement;Map.Entry<Integer, Integer> entry;19;
9;conditional;fun_1();19;
10;conditional;fun_1() == fun_2();19;
11;statement;count++;19;
12;statement;return foo_1;20;
13;exit;;;
Edges
0;2;
1;3;
2;4;
3;5;7;
4;6;
5;4;
6;8;
7;9;
8;10;12;
9;11;9;
10;9;
11;13;
12;

File: cfgs/pythoncfgs/ARRAY_RANGE_QUERIES_ELEMENTS_FREQUENCY_VALUE.py
Nodes
1;statement;foo_1 = fun_1();8;
2;conditional;foo < foo;9;
3;statement;foo_1 = 0;14;
4;conditional;foo_1;15;
5;statement;return foo_1;18;
6;conditional;foo_1 == foo_2[foo_3];16;
7;statement;foo_1 += 1;17;
8;conditional;foo_1[foo_2] in fun_1();10;
9;statement;foo_1[foo_2[foo_3]] = 1;13;
10;statement;foo_1[foo_2[foo_3]] += 1;11;
11;exit;;
Edges
0;2;
1;3;8;
2;4;
3;5;6;
4;11;
5;4;7;
6;4;
7;9;10;
8;2;
9;2;
10;
